x/staking/genesis.go

```diff
@@ -18,9 +17,11 @@ import (
 // Returns final validator set after applying all declaration and delegations
 func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) (res []abci.ValidatorUpdate, err error) {
 
-	// We need to pretend to be "n blocks before genesis", where "n" is the validator update delay,
-	// so that e.g. slashing periods are correctly initialized for the validator set
-	// e.g. with a one-block offset - the first TM block is at height 1, so state updates applied from genesis.json are in block 0.
+	// We need to pretend to be "n blocks before genesis", where "n" is the
+	// validator update delay, so that e.g. slashing periods are correctly
```

Comment:
Due to tombstone, we no longer have slashing periods, correct?

Reply:
I'm not certain of that - I'm pretty sure tombstone actually just implements "infinite slashing periods" - anyways this was just a formatting change not content, so I'm leaving for this PR